raspimouse/src/raspimouse_component.cpp

```diff
@@ -123,8 +125,10 @@ CallbackReturn Raspimouse::on_configure(const rclcpp_lifecycle::State &)
       &Raspimouse::publish_switches, this));
   switches_timer_->cancel();
   // Timer for publishing light sensor information
+  declare_parameter(LIGHT_SENSORS_HZ_PARAM, 10.0);
+  std::chrono::milliseconds ls_duration{static_cast<long int>(1000.0/get_parameter(LIGHT_SENSORS_HZ_PARAM).get_value<double>())};
```

Comment:
変数名を略さないほうが、読みやすいコードになります。

millisecondsの引数には45ビット以上の符号付き整数型が必要なので、`int64_t`でキャストしてください。
https://cpprefjp.github.io/reference/chrono/duration_aliases.html
```suggestion
  std::chrono::milliseconds light_sensors_duration{static_cast<int64_t>(1000.0/get_parameter(LIGHT_SENSORS_HZ_PARAM).get_value<double>())};
```